internal/handler/ticket.go

```diff
@@ -0,0 +1,40 @@
+package handler
+
+import (
+	"log/slog"
+	"net/http"
+
+	"github.com/dilyara4949/flight-booking-api/internal/handler/response"
+	"github.com/dilyara4949/flight-booking-api/internal/middleware"
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+	"golang.org/x/net/context"
+)
+
+type TicketService interface {
+	Delete(ctx context.Context, ticketID, userID uuid.UUID) error
+}
+
+func DeleteTicketHandler(service TicketService) gin.HandlerFunc {
+	return func(c *gin.Context) {
+		ticketID, err := uuid.Parse(c.Param("ticketId"))
```

Comment:
don't forget to perform access check

Reply:
didn't check it since i am taking user id from context, and it will be deleted only if ticket's userId will match id from context:
```go
repo.db.WithContext(ctx).Delete(domain.Ticket{ID: ticketID, UserID: userID})
```

Reply:
How would someone with admin access be able to delete ticket in that case?